faqs.rst

```diff
@@ -20,3 +20,95 @@ How do I report a bug?
 ----------------------
 
 You can read up on reporting bugs here: :ref:`contributing` or report one directly for flux `core <https://github.com/flux-framework/flux-core/issues>`_ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
+
+.. _not_managing_all_resources:
+
+---------------------------------------------------------------------------------
+Why is Flux not discovering and managing all of the resources on the system/node?
+---------------------------------------------------------------------------------
+
+This can be due to various bind flags that need to be passed parallel launcher that started Flux. For example at LLNL you must pass ``--mpibind=off`` to ``srun`` and ``--bind=none`` to ``jsrun``.
+
+Also on all systems, Flux relies on hwloc to auto-detect the on-node resources
+available for scheduling.  The hwloc that Flux is linked against must be
+configured with ``--enable-cuda`` for Flux to be able to detect Nvidia GPUs.
+
+You can test to see if your system default hwloc is CUDA-enabled with:
+
+.. code-block:: sh
+
+  lstopo | grep CoProc
+
+If no output is produced, then your hwloc is not CUDA-enabled.
+
+.. _launch_large_num_jobs:
+
+------------------------------------------------------------
+How do I efficiently launch a large number of jobs to Flux?
+------------------------------------------------------------
+
+See `bulksubmission <https://github.com/flux-framework/flux-workflow-examples/tree/master/async-bulk-job-submit>`_ for an example workflow or flux `tree <https://github.com/flux-framework/flux-sched/blob/master/src/cmd/flux-tree>`_.
+
+.. _node_memory_exhaustion:
+
+------------------------------------------------------------------
+Memory exhaustion on a node when running large ensembles with Flux
+------------------------------------------------------------------
+
+Set ``rundir`` so that the content store is not saved to ``/tmp`` but to filesystem.
+An example command to set the ``rundir`` could look like:
+
+.. code-block:: sh
+
+    flux start -o,-Srundir=/path/to/rundir
+
+.. _mimic_slurm_jobstep:
+
+----------------------------------------
+How do I mimic Slurm's jobstep semantics
+----------------------------------------
+
+``flux mini submit`` ting a script with ``flux mini run`` s will not result in Slurm-style jobsteps unless you stick ``flux-start`` in front of the jobscript in ``flux mini submit`` .
+
+.. _mpi_bootstrap_fails:
+
+----------------------------------------------------------------------------------------------
+Flux is failing to bootstrap a specific MPI implementation (e.g. OpenMPI, MPICH, Spectrum MPI)
+----------------------------------------------------------------------------------------------
+
+Use the ``flux mini run/submit -o mpi=`` option when running or submitting.
```

Comment:
I would add some additional context here as well.  Something along the lines of "Flux's shell plugins for Intel MPI, MVAPICH, and OpenMPI run by default with every job.  If you experience any issues bootstrapping these MPIs, please open an issue (link to the FAQ on submitting issues).  For Spectrum MPI follow the instructions here: LINK"